docs/content/getting-started/deployment/via-clusterctl.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Via clusterctl"
+icon = "fa-solid fa-circle-nodes"
+weight = 1
++++
+
+Add the following to your `clusterctl.yaml` file, which is normally found at
+`${XDG_CONFIG_HOME}/cluster-api/clusterctl.yaml` (or `${HOME}/cluster-api/clusterctl.yaml`). See [clusterctl
+configuration file] for more details. If the `providers` section already exists, add the entry and omit the `providers`
+key from this block below:
+
+```yaml
+providers:
+  - name: "caren"
+    url: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/latest/runtime-extension-components.yaml"
```

Comment:
this is cool solution. do we have a PR on clusterctl cli itself for default location to look for similar to this https://github.com/kubernetes-sigs/cluster-api/blob/main/cmd/clusterctl/client/config/providers_client.go#L277

Reply:
Not yet, but i'm happy to open one.